packages/react-reconciler/src/__tests__/ReactHooksWithNoopRenderer-test.internal.js

```diff
@@ -25,21 +28,27 @@ let useCallback;
 let useMemo;
 let useRef;
 let useImperativeHandle;
+let useTransition;
+let useDeferredValue;
 let forwardRef;
 let memo;
 let act;
 
 describe('ReactHooksWithNoopRenderer', () => {
   beforeEach(() => {
     jest.resetModules();
+    jest.useFakeTimers();
 
     ReactFeatureFlags = require('shared/ReactFeatureFlags');
     ReactFeatureFlags.debugRenderPhaseSideEffectsForStrictMode = false;
     ReactFeatureFlags.enableSchedulerTracing = true;
+    ReactFeatureFlags.flushSuspenseFallbacksInTests = false;
```

Comment:
Why is this necessary? Can we rewrite the test somehow? I can pair with you on this if needed